requirements.txt

```diff
@@ -1,6 +1,7 @@
 numpy
 matplotlib
-scikit-learn==1.1.3
+scikit-learn
```

Comment:
Removal of this restriction will disable dependabot. The new versions of sklearn have not been compatible (see #71).

Reply:
Version was fixed here, I don't remember which toolbox was incompatible: https://github.com/wwu-mmll/photonai/commit/77436e355cb500ccf87ab4633978109dc7a5fc91